docs/blog/_posts/2019-01-17-12th-dotty-milestone-release.md

```diff
@@ -0,0 +1,197 @@
+---
+layout: blog-page
+title: Announcing Dotty 0.12.0-RC1
+author: Aggelos Biboudis
+authorImg: /images/aggelos.jpg
+date: 2019-01-17
+---
+
+Happy New Year to all with the first release of Dotty for 2019! ✨🎊🎉
+
+Today we are excited to release the version 0.12.0-RC1 of the Dotty compiler. 
+This release serves as a technology preview that demonstrates new language features and the
+compiler supporting them.
+
+Dotty is the project name for technologies that are considered for inclusion in Scala 3. Scala has
+pioneered the fusion of object-oriented and functional programming in a typed setting. Scala 3 will
+be a big step towards realising the full potential of these ideas. Its main objectives are to
+
+- become more opinionated by promoting programming idioms we found to work well,
+- simplify where possible,
+- eliminate inconsistencies and surprising behaviours,
+- build on strong foundations to ensure the design hangs well together,
+- consolidate language constructs to improve the language’s consistency, safety, ergonomics, and
+  performance.
+
+You can learn more about Dotty on our [website](https://dotty.epfl.ch).
+
+<!--more-->
+
+This is our 12th scheduled release according to our
+[6-week release schedule](https://dotty.epfl.ch/docs/usage/version-numbers.html).
+
+## What’s new in the 0.12.0-RC1 technology preview?
+
+### Extension Methods
+
+We are excited to announce that extension methods are now offered through dedicated language support!

```

Comment:
I think it's important to note that:
- Extension methods have not gone through the SIP process yet
- In particular, the syntax might still change (and link to the PR where most of the discussion took place so far)

Reply:
Resolving since everything will go through the SIP process eventually. Thx!